app/models/role.rb

```diff
@@ -80,52 +143,17 @@ def access_level
     end
   end
 
-  # Sets access_level according to bit fields defined in the column access
-  # TODO refactor according to the hash defined above (e.g. 1 key is :creator, 2 key is :administrator, etc)
-  def set_access_level(access_level)
```

Comment:
Deprecated this method and replaced with an idiomatic Ruby setter method.

Reply:
that's much cleaner